cpp/open3d/visualization/visualizer/O3DVisualizer.cpp

```diff
@@ -1706,6 +1711,14 @@ Ctrl-alt-click to polygon select)";
         }
     }
 
+    void SetCollapsableVertOpen(const std::string &name, bool open) {
```

Comment:
nit: `SetPanelOpen` is clearer for users. O3DVisualizer is meant to be more high level, so referring to the widgets as panels is easier for a new user to find than CollapsableVert.

Reply:
Updated in commit https://github.com/isl-org/Open3D/pull/6865/commits/a19cdf12bc6329a88047f2c602175005f7267eaf